apps/io-func-sign-user/src/infra/http/encoders/signature-request.ts

```diff
@@ -90,6 +89,13 @@ export const SignatureRequestToApiModel: E.Encoder<
           signed_at: extra.signedAt,
         };
       }
+      case "CANCELED": {
+        return {
+          ...commonFields,
+          status: SignatureRequestStatusEnum.REJECTED,
+          reject_at: extra.canceledAt,
+        };
+      }
```

Comment:
The user side must also have the canceled status
```suggestion
      case "CANCELED": {
        return {
          ...commonFields,
          status: SignatureRequestStatusEnum.CANCELLED,
          cancelled_at: extra.canceledAt,
        };
      }
```

Reply:
sure! it was a mistake from copy and paste, thank you!